scripts/ci/forbidden-identifiers.js

```diff
@@ -47,69 +46,105 @@ findTestFiles()
+ */
+function findErrors(diffList) {
+  return diffList.reduce((errors, diffFile) => {
+    let fileName = diffFile.name;
+    let content = diffFile.content.split('\n');
+    let lineNumber = 0;
 
-        let matches = line.match(blockedRegex);
-        let scopeImport = path.relative(libRoot, fileName).startsWith('..')
-          ? isRelativeScopeImport(fileName, line)
-          : false;
+    content.forEach(line => {
+      lineNumber++;
 
-        if (matches || scopeImport) {
+      let matches = line.match(blockedRegex);
 
-          let error = {
-            fileName: fileName,
-            lineNumber: lineCount,
-            statement: matches && matches[0] || scopeImport.invalidStatement
-          };
+      if (matches) {
 
-          if (scopeImport) {
-            error.messages = [
-              'You are using an import statement, which imports a file from another scope package.',
-              `Please import the file by using the following path: ${scopeImport.scopeImportPath}`
-            ];
-          }
+        errors.push({
+          fileName,
+          lineNumber,
+          statement: matches[0]
+        });
 
-          errors.push(error);
-        }
-      });
+      }
+    });
 
-      return errors;
+    return errors;
 
-    }, []);
-  })
+  }, []);
+}
 
-  /* Print the resolved errors to the console */
-  .then(errors => {
-    if (errors.length > 0) {
-      console.error('Error: You are using one or more blocked statements:\n');
+/**
+ * Groups similar errors in the same file which are present over a range of lines.
+ * @param {{fileName: string, lineNumber: number, statement: string}[]} errors
+ */
+function groupErrors(errors) {
 
-      errors.forEach(entry => {
-        if (entry.messages) {
-          entry.messages.forEach(message => console.error(`   ${message}`))
-        }
+  let initialError, initialLine, previousLine;
 
-        console.error(`   ${entry.fileName}@${entry.lineNumber}, Statement: ${entry.statement}.\n`);
-      });
+  return errors.filter(error => {
 
-      process.exit(1);
+    if (initialError && isSimilarError(error)) {
+      previousLine = error.lineNumber;
+
+      // Overwrite the lineNumber with a string, which indicates the range of lines.
+      initialError.lineNumber = `${initialLine}-${previousLine}`;
+
+      return false;
     }
-  })
 
-  .catch(err => {
-    // An error occurred in this script. Output the error and the stack.
-    console.error('An error occurred during execution:');
-    console.error(err.stack || err);
-    process.exit(2);
+    initialLine = previousLine = error.lineNumber;
+    initialError = error;
+
+    return true;
   });
 
+  /** Checks whether the given error is similar to the previous one. */
+  function isSimilarError(error) {
+    return initialError.fileName === error.fileName &&
+           initialError.statement === error.statement &&
+           previousLine === (error.lineNumber - 1);
+  }
```

Comment:
I know this is not very pretty, but we don't have any Lodash helpers here, and this seems to be an elegant approach.